src/components/ui/Button/LinkButton.tsx

```diff
@@ -0,0 +1,44 @@
+import type { AnchorHTMLAttributes, ReactNode } from 'react'
+import React from 'react'
+import { Icon as UIIcon, Link } from '@faststore/ui'
+
+import './buttons.scss'
+
+type Variant = 'primary' | 'secondary'
+type IconPosition = 'left' | 'right'
+
+type Props = {
+  variant?: Variant
+  inverse?: boolean
+  icon?: ReactNode
+  iconPosition?: IconPosition
+  disabled?: boolean
+} & AnchorHTMLAttributes<HTMLAnchorElement>
+
+function LinkButton({
+  variant,
+  inverse,
+  icon,
+  iconPosition,
+  children,
+  disabled = false,
+  className,
```

Comment:
Do you mean other classes that this component will recieve? If so, I think it's better to use `classes` instead of `classNames`. What do you think?

Reply:
i think this component can receive other classes, if a user needs to do a specific style, why not className?  Wouldn't it be better to stick with naming pattern?

Reply:
hmm, got it! I'm just wondering if it's going to be confusing, but I'm fine with that.